src/cljs/zetawar/subs.cljs

```diff
@@ -456,8 +455,9 @@
 
 (deftrack compatible-armor-types-for-repair? [conn targeted-q targeted-r]
   (when-let [[selected-q selected-r] @(selected-hex conn)]
-    (game/compatible-armor-types-for-repair? @conn @(game conn) @(unit-at conn selected-q selected-r)
-                                                                @(unit-at conn targeted-q targeted-r))))
+    (game/compatible-armor-types-for-repair? @conn @(game conn)
```

Comment:
I made the changes and the tests passed.
Should I move the compatible-armor-types-for-repair? track out of the "Selection and target" section and into the "Units" section? It uses the selected-hex track and calls a function from game.cljs, so it's kind of in-between. Maybe break it up into two tracks?